delivery_schenker/views/stock_picking_views.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <!-- Copyright 2021 Tecnativa - David Vidal
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).-->
+    <record id="view_picking_withcarrier_out_form" model="ir.ui.view">
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="delivery.view_picking_withcarrier_out_form" />
+        <field name="groups_id" eval="[(4, ref('base.group_no_one'))]" />
+        <field name="arch" type="xml">
+            <xpath expr="//header" position='inside'>
+                <button
+                    name="schenker_get_label"
+                    string="Schenker Label"
+                    type="object"
+                    attrs="{'invisible':['|', '|', ('delivery_type', '!=', 'schenker'), ('carrier_tracking_ref', '=', False), ('state', '!=', 'done')]}"
+                />
+            </xpath>
+        </field>
+    </record>
+</odoo>
```

Comment:
@chienandalu This slipped through due to changes on an existing DB I imagine, the `schenker_get_label` function doesn't exist on `stock.picking` and it stops the module from even installing. We would need an extra function on `stock.picking` for this which calls the same function on `delivery.carrier` and adds the PDF to attachments or posts it in the chatter

Reply:
Sure. That was meant to work with the tracking features. It should go away until they're implemented. I'll patch it tomorrow

Reply:
Fix here: https://github.com/OCA/delivery-carrier/pull/447